README.md

```diff
@@ -5,7 +5,7 @@
 # cvxpylayers
```

Comment:
At some point we should update the CVXPY Layers logo to include JAX :)

Reply:
Done :)